apps/condo/package.json

```diff
@@ -91,6 +91,7 @@
     "body-parser": "^1.19.1",
     "carbone": "3.2.3",
     "classnames": "^2.2.6",
+    "csv-parse": "^5.5.6",
```

Comment:
It is not a version, that used inside another apps!

Reply:
Fixed